src/internet_identity/internet_identity.did

```diff
@@ -74,6 +74,16 @@ type DeviceData = record {
     protection: DeviceProtection;
 };
 
+type ReadOnlyDeviceData = record {
```

Comment:
that's quite a long name! why is it RO btw? I guess there's no "RW" devices, but some are "read" and some are "write"

Also, can you add a note about why this differs from the other device type? That's not immediately clear to me.

Finally, I've often seen apis wrap a type with e.g. `type Timestamped<T> = { dat: T, timestamp: Timestamp }`. That might help deduplicate things!

Reply:
Making `IdentityAnchorInfo.devices` a vector of `TimestampedDevice = record { dat: DeviceData, timestamp: Timestamp }` would be a breaking change. I'd rather avoid that.